docs/api/Agent.md

```diff
@@ -2,12 +2,18 @@
 
 ## `new undici.Agent(opts)`
 
-* opts `undici.Pool.options` - options passed through to Pool constructor
+* opts `AgentOptions` - options passed through to Pool constructor
 
 Returns: `Agent`
 
 Returns a new Agent instance for use with pool based requests or the following top-level methods `request`, `pipeline`, and `stream`.
 
+### Parameter: `AgentOptions`
+
+Extends: [`PoolOptions`](docs/api/Pool.md#parameter-pooloptions)
+
+* **factory** `(url: string, options?: Client.Options): Pool` (optional) - A factory which returns the pool to use for the request.
```

Comment:
Can you add an example on how to set up an agent to follow redirects? Something copy-pastable.

Reply:
Sure. No problem.

Reply:
Done!